src/gui/socketapi.cpp

```diff
@@ -195,7 +202,16 @@ SocketApi::SocketApi(QObject *parent)
 #endif
     } else if (Utility::isLinux() || Utility::isBSD()) {
         QString runtimeDir;
+#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
```

Comment:
I think meanwhile we do check this on the lowest level anyway, no? That said, I think we shouldn't introduce these checks (as they assume we do it for every function that needs a newer version, which is clearly not the case for the whole code base). Instead, this should be left to specific patches that address compatibility issues for branches that build for older distros (e.g. the Ubuntu ppas)